### PR TITLE
fix: add locale parameter for isAlpha and isAlphanumeric validators

### PR DESCRIPTION
**Pull request by rubiin**
This commit adds the feature to add locale to both isAlpha and isAlphanumeric with default as en-US

### CODE REVIEW

File: src/validation/Validator.ts [hunk 1]
```diff
@@ -495,16 +495,16 @@ export class Validator {
      * Checks if the string contains only letters (a-zA-Z).
      * If given value is not a string, then it returns false.
      */
-    isAlpha(value: string): boolean {
-        return typeof value === "string" && this.validatorJs.isAlpha(value);
+    isAlpha(value: string, locale: ValidatorJS.AlphaLocale = "en-US"): boolean {
```
Comment (vlapo) [inline src/validation/Validator.ts]: Please leave locale parameter without default value. `validator.js` lib will use "en-US" by default. Same for isAlphanumeric.

**Comment by l2ubin** [comment]
will do

**Comment by l2ubin** [comment]
I have also fixed the number options on the isnumeric string on the same pr

**Comment by l2ubin** [comment]
@vlapo I have addressed your changes.

**Comment by vlapo** [comment]
Thank you for contribution.

**Comment by rubiin** [comment]
Anytime

**Pull request merged**